### PR TITLE
Buckets Now Reflect When They're (Mostly) Full of Liquid

### PR DESCRIPTION
**Pull request by Captain277**
## About The Pull Request

1. **Enables Overlay that Shows When a Bucket Has Liquid In It.**

## Why It's Good For The Game

1. _Just a nice little QOL thing. Or really just an aesthetic thing, really._

## Changelog

:cl:
tweak: Enables check for water overlay on buckets.
/:cl:

### CODE REVIEW

File: code/modules/reagents/reagent_containers/glass.dm [hunk 1]
```diff
@@ -299,11 +299,18 @@
...
+	update_icon()
+
 /obj/item/reagent_containers/glass/bucket/update_icon()
 	cut_overlays()
 	if (!is_open_container())
 		var/image/lid = image(icon, src, "lid_[initial(icon_state)]")
 		add_overlay(lid)
+	if (is_open_container() && reagents.total_volume > 100)
+		var/image/water = image(icon, src, "water_[initial(icon_state)]")
+		add_overlay(water)
```
Comment (TheLordME) [inline code/modules/reagents/reagent_containers/glass.dm:313]: does this overlay work with the wooden version below?

Comment (TheLordME) [reply code/modules/reagents/reagent_containers/glass.dm:313]: if not, please either make it work, or make sure its not applied

**Review by TheLordME — CHANGES_REQUESTED** [review]

File: code/modules/reagents/reagent_containers/glass.dm (hunk 1 above)
Comment (Captain277) [reply code/modules/reagents/reagent_containers/glass.dm:313]: Yeah it should work globally.

Comment (Captain277) [reply code/modules/reagents/reagent_containers/glass.dm:313]: Being that the wood and standstone buckets are still both subtypes of glass/bucket.

Comment (Captain277) [reply code/modules/reagents/reagent_containers/glass.dm:313]: Whoops I stand corrected. Apparently it's not asserting. Easy fix.

Comment (TheLordME) [reply code/modules/reagents/reagent_containers/glass.dm:313]: my main concern is that the overlay does not fit visually into the bucket

**Review by TheLordME — APPROVED** [review]

**Pull request merged**